objects/puzzles.rb

```diff
@@ -78,6 +78,16 @@ def group(xml)
   # Take some puzzles from the XML and either close their tickets in GitHub
   # or create new tickets.
   def expose(xml, tickets)
+    close(xml, tickets)
+    skip_model = xml.xpath('/puzzles[@model="true"]').empty?
```

Comment:
@mbao01 I think this flag should be taken from `.0pdd` configuration file, not from XML